src/components/ProposalListGridPanel.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import {
+  Panel,
+  PanelBody,
+} from './Panel';
+import { ProposalListGrid } from './ProposalListGrid';
+
+interface ProposalListGridPanelProps {
+  proposals: {
+    id: string;
+    values: Record<string, string[]>;
+  }[];
+}
+
+export const ProposalListGridPanel = ({ proposals }: ProposalListGridPanelProps) => (
```

Comment:
> To add it to a page, ...

Would you like me to wire it up as part of this PR, or shall we merge this and I'll open a follow-up?

Reply:
Since the page we want it on isn't populating the `proposals` yet, I figured we'd wait until then to actually add it.